src/Control/Parallel.purs

```diff
@@ -48,3 +50,13 @@ parSequence_
   => t (m a)
   -> m Unit
 parSequence_ = parTraverse_ id
+
+parOneOf
+  :: forall a t m f
+   . Parallel f m
+  => Plus f
```

Comment:
Yeah, I guessed so, but I'm wondering if it's helpful to have the generality. We'd need something which has `Plus` and `Applicative` but not `Alternative`. If we assume `Alternative`, then we get two extra laws that we can use to think about how this function interoperates with `<*>`.